substrate/primitives/core/src/crypto.rs

```diff
@@ -593,13 +593,21 @@ impl std::fmt::Display for AccountId32 {
 }
 
 impl sp_std::fmt::Debug for AccountId32 {
-	#[cfg(feature = "std")]
+	#[cfg(any(feature = "std", debug_assertions))]
```

Comment:
does this also work?

```suggestion
	#[cfg(any(feature = "std", debug_assertions, test))]
```

Reply:
that should not be necessary ? 
All tests with the exception of integration and benchmark should run with std and debug_assertions on.